<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>711</width>
      <height>481</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1. a/(b-c)*(d+e)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2. (a+b)*(c-d)/e</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3. a-(b+c*d)/e</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4. a/b-((c+d)*e)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5. a*(b-c+d)/e</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6. (a+b)*(c-d)/e</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7. a*(b-c)/(d+e)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8. a/(b*(c+d))-e</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9. (a+(b/c-d))*e</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10. a*(b+c)/(d-e)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11. a-(b/c*(d+e))</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12. (a-b)/(c+d)*e</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13. a/(b+c-d*e)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14. a*(b-c)/(d+e)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>15. (a+b*c)/(d-e)</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>a</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>b</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>c</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>d</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>e</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Результат</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>8.6</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>2.4</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>5.1</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>0.3</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>7.9</string>
     </property>
    </item>
    <item row="0" column="5">
     <property name="text">
      <string>-26.12</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>7.4</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>3.6</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>2.8</string>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string>9.5</string>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string>0.9</string>
     </property>
    </item>
    <item row="1" column="5">
     <property name="text">
      <string>-81.89</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>3.1</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>5.4</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>0.2</string>
     </property>
    </item>
    <item row="2" column="3">
     <property name="text">
      <string>9.6</string>
     </property>
    </item>
    <item row="2" column="4">
     <property name="text">
      <string>7.8</string>
     </property>
    </item>
    <item row="2" column="5">
     <property name="text">
      <string>2.16</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>1.2</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>0.7</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>9.3</string>
     </property>
    </item>
    <item row="3" column="3">
     <property name="text">
      <string>6.5</string>
     </property>
    </item>
    <item row="3" column="4">
     <property name="text">
      <string>8.4</string>
     </property>
    </item>
    <item row="3" column="5">
     <property name="text">
      <string>-131.006</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>9.7</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>8.2</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>3.6</string>
     </property>
    </item>
    <item row="4" column="3">
     <property name="text">
      <string>4.1</string>
     </property>
    </item>
    <item row="4" column="4">
     <property name="text">
      <string>0.5</string>
     </property>
    </item>
    <item row="4" column="5">
     <property name="text">
      <string>168.78</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>0.8</string>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string>4.1</string>
     </property>
    </item>
    <item row="5" column="2">
     <property name="text">
      <string>7.9</string>
     </property>
    </item>
    <item row="5" column="3">
     <property name="text">
      <string>6.2</string>
     </property>
    </item>
    <item row="5" column="4">
     <property name="text">
      <string>3.5</string>
     </property>
    </item>
    <item row="5" column="5">
     <property name="text">
      <string>2.38</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>1.6</string>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string>4.9</string>
     </property>
    </item>
    <item row="6" column="2">
     <property name="text">
      <string>5.7</string>
     </property>
    </item>
    <item row="6" column="3">
     <property name="text">
      <string>0.8</string>
     </property>
    </item>
    <item row="6" column="4">
     <property name="text">
      <string>2.3</string>
     </property>
    </item>
    <item row="6" column="5">
     <property name="text">
      <string>-0.413</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>8.5</string>
     </property>
    </item>
    <item row="7" column="1">
     <property name="text">
      <string>0.3</string>
     </property>
    </item>
    <item row="7" column="2">
     <property name="text">
      <string>2.4</string>
     </property>
    </item>
    <item row="7" column="3">
     <property name="text">
      <string>7.9</string>
     </property>
    </item>
    <item row="7" column="4">
     <property name="text">
      <string>1.6</string>
     </property>
    </item>
    <item row="7" column="5">
     <property name="text">
      <string>1.151</string>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string>5.6</string>
     </property>
    </item>
    <item row="8" column="1">
     <property name="text">
      <string>7.4</string>
     </property>
    </item>
    <item row="8" column="2">
     <property name="text">
      <string>8.9</string>
     </property>
    </item>
    <item row="8" column="3">
     <property name="text">
      <string>3.1</string>
     </property>
    </item>
    <item row="8" column="4">
     <property name="text">
      <string>0.2</string>
     </property>
    </item>
    <item row="8" column="5">
     <property name="text">
      <string>0.666</string>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string>0.4</string>
     </property>
    </item>
    <item row="9" column="1">
     <property name="text">
      <string>2.3</string>
     </property>
    </item>
    <item row="9" column="2">
     <property name="text">
      <string>6.7</string>
     </property>
    </item>
    <item row="9" column="3">
     <property name="text">
      <string>5.8</string>
     </property>
    </item>
    <item row="9" column="4">
     <property name="text">
      <string>9.1</string>
     </property>
    </item>
    <item row="9" column="5">
     <property name="text">
      <string>-1.091</string>
     </property>
    </item>
    <item row="10" column="0">
     <property name="text">
      <string>5.6</string>
     </property>
    </item>
    <item row="10" column="1">
     <property name="text">
      <string>3.2</string>
     </property>
    </item>
    <item row="10" column="2">
     <property name="text">
      <string>0.9</string>
     </property>
    </item>
    <item row="10" column="3">
     <property name="text">
      <string>1.7</string>
     </property>
    </item>
    <item row="10" column="4">
     <property name="text">
      <string>4.8</string>
     </property>
    </item>
    <item row="10" column="5">
     <property name="text">
      <string>-17.51</string>
     </property>
    </item>
    <item row="11" column="0">
     <property name="text">
      <string>0.3</string>
     </property>
    </item>
    <item row="11" column="1">
     <property name="text">
      <string>6.7</string>
     </property>
    </item>
    <item row="11" column="2">
     <property name="text">
      <string>8.4</string>
     </property>
    </item>
    <item row="11" column="3">
     <property name="text">
      <string>9.5</string>
     </property>
    </item>
    <item row="11" column="4">
     <property name="text">
      <string>1.2</string>
     </property>
    </item>
    <item row="11" column="5">
     <property name="text">
      <string>-0.429</string>
     </property>
    </item>
    <item row="12" column="0">
     <property name="text">
      <string>7.6</string>
     </property>
    </item>
    <item row="12" column="1">
     <property name="text">
      <string>4.8</string>
     </property>
    </item>
    <item row="12" column="2">
     <property name="text">
      <string>3.5</string>
     </property>
    </item>
    <item row="12" column="3">
     <property name="text">
      <string>9.1</string>
     </property>
    </item>
    <item row="12" column="4">
     <property name="text">
      <string>0.2</string>
     </property>
    </item>
    <item row="12" column="5">
     <property name="text">
      <string>1.173</string>
     </property>
    </item>
    <item row="13" column="0">
     <property name="text">
      <string>0.5</string>
     </property>
    </item>
    <item row="13" column="1">
     <property name="text">
      <string>6.1</string>
     </property>
    </item>
    <item row="13" column="2">
     <property name="text">
      <string>8.9</string>
     </property>
    </item>
    <item row="13" column="3">
     <property name="text">
      <string>2.4</string>
     </property>
    </item>
    <item row="13" column="4">
     <property name="text">
      <string>7.3</string>
     </property>
    </item>
    <item row="13" column="5">
     <property name="text">
      <string>-0.144</string>
     </property>
    </item>
    <item row="14" column="0">
     <property name="text">
      <string>9.1</string>
     </property>
    </item>
    <item row="14" column="1">
     <property name="text">
      <string>0.6</string>
     </property>
    </item>
    <item row="14" column="2">
     <property name="text">
      <string>2.4</string>
     </property>
    </item>
    <item row="14" column="3">
     <property name="text">
      <string>3.7</string>
     </property>
    </item>
    <item row="14" column="4">
     <property name="text">
      <string>8.5</string>
     </property>
    </item>
    <item row="14" column="5">
     <property name="text">
      <string>-2.196</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Свой тест</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>570</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Тут вывод что получилось в ОПЗ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>590</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выражение c числами</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>570</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Тут результат теста</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>30</y>
      <width>95</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Teст 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Teст 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Teст 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Teст 4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>Teст 5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>Teст 6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>Teст 7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>Teст 8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_9">
       <property name="text">
        <string>Teст 9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_10">
       <property name="text">
        <string>Teст 10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_12">
       <property name="text">
        <string>Teст 11</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_11">
       <property name="text">
        <string>Teст 12</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_13">
       <property name="text">
        <string>Teст 13</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_14">
       <property name="text">
        <string>Teст 14</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_15">
       <property name="text">
        <string>Teст 15</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
